SofaKernel/modules/SofaBaseMechanics/MechanicalObject.h

```diff
@@ -422,8 +417,11 @@ protected :
 
     /// @}
 
-    sofa::core::objectmodel::DataFileName filename;
-    Data< bool> ignoreLoader;
+    //int vsize; ///< Number of elements to allocate in vectors
+    Data< int > d_size;
```

Comment:
ok.
Be aware of the potential conflict with `if (arg->getAttribute("size")` in `MechanicalObject::parse`.